client/src/components/profile/components/time-line.tsx

```diff
@@ -242,6 +242,7 @@ function useIdToNameMap(t: TFunction): Map<string, NameMap> {
                 blockName
               }
               id
+              superBlock
```

Comment:
```suggestion
              superBlock
              hasEditableBoundaries
```
We should use this to determine if the challenge is step or not.

Reply:
Thanks for your review, @ojeytonwilliams.

And sorry, I wasn't aware of the `hasEditableBoundaries` graphQL parameter in spite of knowing that the prop with the same name is used in codebase.

Furthermore, after checking with GraphiQL, I just realized that even if `Step` is included in the `title` parameter, like `Problem 178: Step Numbers`, there is a situation where using `t('aria.step')` for determining if the challenge is step or not is not appropriate.

So I agree with you and commit the suggestions from here since I've tested the code in some language locally.

Reply:
> And sorry, I wasn't aware of the hasEditableBoundaries graphQL parameter in spite of knowing that the prop with the same name is used in codebase.

That's totally fine. It's the job of reviews to catch things like this.

> Furthermore, after checking with GraphiQL, I just realized that even if Step is included in the title parameter, like Problem 178: Step Numbers, there is a situation where using t('aria.step') for determining if the challenge is step or not is not appropriate.

Yeah, that's a good example.